Orange/widgets/data/owpythonscript.py

```diff
@@ -513,25 +520,36 @@ def __init__(self):
         self.controlArea.layout().addStretch(1)
         self.resize(800, 600)
 
+    def _set_dict(self, dic, input, id):
+        id = id[0] if isinstance(id, dict) else id
```

Comment:
Can id be a dict? (And why would this dict have a key `0`?) As I can see, it can be a tuple and it's OK if it remains a tuple.